pkg/kncloudevents/good_client.go

```diff
@@ -20,6 +20,22 @@ type ConnectionArgs struct {
 }
 
 func NewDefaultClient(target ...string) (cloudevents.Client, error) {
+	t, err := newHTTPTransport(target...)
+	if err != nil {
+		return nil, err
+	}
+	return NewDefaultClientGivenHttpTransport(t)
+}
+
+func NewDefaultClientGivenConnectionArgs(connectionArgs ConnectionArgs, target ...string) (cloudevents.Client, error) {
```

Comment:
can we use `NewDefaultClientGivenHttpTransport` or we need this extra wrapper?

Reply:
You are right, we can use the `NewDefaultClientGivenHttpTransport` function.
Fixed.